articles/openshift/oshift-netpol.md

```diff
@@ -0,0 +1,138 @@
+---
+title: How to use NetworkPolicy objects to connect services between projects | UKCloud Ltd
+description: Shows how to create a basic NetworkPolicy object to connect services in two projects
+services: openshift
+author: Kieran O'Neill
+
+toc_rootlink: How To
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Using network policy
+toc_fullpath: How To/oshift-netpol.md
+toc_mdlink: oshift-netpol.md
+---
+
+# How to use NetworkPolicy objects to connect services between projects
+
+## Overview
+
+In v3.11 clusters, deployed from mid-February 2019, the default SDN plugin has been changed from ovs-multitenant to ovs-networkpolicy. This gives you the ability to create NetworkPolicy objects which allow granular control over the flow of communication between pods/services/projects inside your cluster. 
+
+By default we put in place NetworkPolicy objects that allow all pods to communicate with each other inside the same project and allow communication from all pods/services to the default and openshift-monitoring projects. This mirrors the default pod security provided with the ovs-multitenant plugin. One of these objects will exist in all projects, including projects created after the cluster is deployed and have the name 'allow-from-same-and-privileged-namespaces'
+
+## Interacting with NetworkPolicy objects
+
+NetworkPolicy objects can be interacted with on the command line using the full name networkpolicy or short name netpol, for example:
```

Comment:
You can interact with NetworkPolicy object from the command line using the full name `networkpolicy` or short name `netpol`, for example:

Reply:
Fixed typo from me